docs/DeveloperGuide.md

```diff
@@ -184,7 +185,7 @@ detailed view:
 
 * **Alternative 1 (current choice):** Use JavaFX ListView
   * Pros: Easy to keep UI up to sync with model by overriding ListCell's updateItem method
-  * Cons: Can allow for displaying of multiple DetailedBookCards even though the detailed view is only meant to show
+  * Cons: Technically allows for displaying of multiple books even though the detailed view is only meant to show
```

Comment:
I think saying 'Technically Allows' is a bit weird because it implies that it is a bug. Maybe we can add a statement to explain why the bug wouldn't occur?